_rules/SC4-1-2-iframe-unique-name.md

```diff
@@ -0,0 +1,294 @@
+---
+name: Iframes with identical accessible names serve equivalent purpose
+
+description: |
+ This rule checks that iframes with identical accessible names serve an equivalent purpose
+
+success_criterion:
+- 4.1.2
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Jey Nandakumar
+- Audrey Maniez
+---
+
+## Test procedure
+
+### Applicability
+
+This rule applies to `iframe` elements that are [included in the accessibility tree](#included-in-the-accessibility-tree) and have [matching](matching-characters) [accessible name](#accessible-name), that do not only include whitespace.
```

Comment:
I still suggest aligning with https://github.com/auto-wcag/auto-wcag/pull/220, which would mean this applicability:

> This rule applies to any set of any two or more `iframe` elements in the same [document tree](https://www.w3.org/TR/dom41/#document-trees) that are [included in the accessibility tree](#included-in-the-accessibility-tree), and that have [matching](#matching-characters) [accessible names](#accessible-name) that does not only consist of [whitespace](#whitespace).
>
> **Note:** The test target for this rule is the full set of `iframe` elements within the same [document tree](https://www.w3.org/TR/dom41/#document-trees) that share the same [matching](#matching-characters) [accessible name](#accessible-name).

Feedback that I got about the other rule was that it is very important to state how to compare the elements - is it comparing them one to one, so that two elements that violate the rule = two failures, one for each element, or is it one failure because the not all elements in the set have the same purpose.
If there are let's say 20 iframes on a page that have the same name, comparing each of them to each of the other ones are quite different from looking at the set at once.

"whitespace" needs to link to the definition anyway.